cluster-autoscaler/core/scale_up.go

```diff
@@ -400,7 +400,7 @@ func ScaleUp(context *context.AutoscalingContext, processors *ca_processors.Auto
 
 		for pod, err := range podsNotPassing {
 			_, found := podsRemainUnschedulable[pod]
-			if found && nodeGroup.Exist() {
+			if found {
```

Comment:
The comment below no longer makes sense with your change.

Reply:
Right, deleted!